test/jsonapi/plugs/query_parser_test.exs

```diff
@@ -14,7 +14,7 @@ defmodule JSONAPI.QueryParserTest do
       [
         author: JSONAPI.QueryParserTest.UserView,
         comments: JSONAPI.QueryParserTest.CommentView,
-        best_friends: JSONAPI.QueryParsertTest.UserView
+        best_friends: JSONAPI.QueryParserTest.UserView
```

Comment:
Nice catch! How is this not causing any issues in our test suite?

Reply:
I don’t believe the code is interpreted until accessed?  (It started to complain / fail the test suite once I fixed the underlying bug)

Reply:
Module names are just atoms, right? So that means you can just type any module name and Elixir is okay. I found this to be a source of frustration when learning Elixir. Here's a surprising example from an IEx session:

```
~ % iex
Erlang/OTP 21 [erts-10.3.4] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:1] [hipe] [dtrace]

Interactive Elixir (1.8.1) - press Ctrl+C to exit (type h() ENTER for help)
iex(1)> Foo.Bar.Baz
Foo.Bar.Baz
```

Boom. Totally okay.

Reply:
That explains it!  Thanks!